packages/commonwealth/server/workers/commonwealthConsumer/policies/chainEventCreated/chainEventCreatedPolicy.ts

```diff
@@ -17,6 +19,13 @@ export const processChainEventCreated: EventHandler<
     payload.eventSource.eventSignature === communityStakeTradeEventSignature
   ) {
     await handleCommunityStakeTrades(models, payload);
+  } else if (
+    payload.eventSource.eventSignature === launchpadTokenLaunchedEventSignature
+  ) {
+    await Token.createTokenHandler(
```

Comment:
This is probably not ideal, but we don't have a trpc query to get ChainNodes. As a result I didn't want to have to add one. Instead I just created this helper which is the core processing logic for the createTokenHandler. The models are still kept only in the command.

I should refactor it and add a query for chain nodes, but it might bloat this PR too much